pkg/reconciler/autoscaling/kpa/kpa_test.go

```diff
@@ -126,23 +126,14 @@ func initialScaleZeroASConfig() *autoscalerconfig.Config {
 	return ac
 }
 
-func activatorCertsNetConfig() *netcfg.Config {
-	nc, _ := netcfg.NewConfigFromMap(map[string]string{
-		netcfg.ActivatorCAKey:  "knative-ca",
-		netcfg.ActivatorSANKey: "knative-san",
-	})
-	return nc
-}
-
 func defaultConfig() *config.Config {
 	ac, _ := asconfig.NewConfigFromMap(defaultConfigMapData())
 	deploymentConfig, _ := deployment.NewConfigFromMap(map[string]string{
 		deployment.QueueSidecarImageKey: "bob",
 		deployment.ProgressDeadlineKey:  progressDeadline.String(),
 	})
 	networkConfig, _ := netcfg.NewConfigFromMap(map[string]string{
-		netcfg.ActivatorCAKey:  "",
-		netcfg.ActivatorSANKey: "",
+		netcfg.InternalEncryptionKey: "false",
```

Comment:
Default for that flag is false anyway it does not add much here. `networkConfig, _ := netcfg.NewConfigFromMap(nil)` also works. We only need to return the default config here.